Samples/samples.json

```diff
@@ -491,33 +491,51 @@
     },
     {
       "owner": "milismsft",
-      "filePath": "Samples/Compute/ManageContainerServiceUsingKubernetes.cs",
-      "githubPath": "Azure-Samples/acs-dotnet-manage-azure-container-service",
```

Comment:
Do I need to delete this repository?

Reply:
yes please; we did the same for the Java release, right?